### PR TITLE
Add option type and improve Storybook

### PR DESCRIPTION
**Pull request by rkuykendall**
Closes https://github.com/mighty-justice/fields-ant/issues/36

image

### CODE REVIEW

File: .storybook/stories.css [hunk 1]
```diff
@@ -0,0 +1,14 @@
+.story-component {
```
Comment (alexandermeyer) [inline .storybook/stories.css]: This looks cumbersome, did you write this style yourself? Is there a before after?

File: src/inputs/RadioGroup.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    if (this.injected.getOptions) { return this.injected.getOptions(optionType); }
+    return [];
+  }
+
+  private get options (): IOption[] {
+    return this.fieldConfig.sorted ? sortBy(this.unsortedOptions, 'name') : this.unsortedOptions;
+  }
+
+  public render () {
+    const radioStyle = {
```
Comment (alexandermeyer) [inline src/inputs/RadioGroup.tsx]: You can define this outside of the class definition. It doesn't look like we have style files for our components in this repository yet, is that on the docket?

File: .storybook/config.js [hunk 3]
```diff
@@ -1,16 +1,70 @@
...
+  ],
+};
+
+const ComponentDecorator = (storyFn) => (
+    <div className='story-component'>
+      { storyFn() }
+    </div>
+);
+
+addDecorator(ComponentDecorator);
```
Comment (alexandermeyer) [inline .storybook/config.js]: Why do you need two decorators that both call `storyFn()`? I'm not familiar with storybook

Comment (rkuykendall) [reply .storybook/config.js]: The decorators stack and wrap each other. So one needs to be before the withInfo, and one after.

File: .storybook/stories.css (hunk 1 above)
Comment (rkuykendall) [reply .storybook/stories.css]: Added a comment. This adds the blueprint background behind components.

File: src/inputs/RadioGroup.tsx (hunk 2 above)
Comment (rkuykendall) [reply src/inputs/RadioGroup.tsx]: No, I think I'll just remove it here so it can be styled on-site like all our other components.

**Comment by rkuykendall** [comment]
@alexandermeyer I considered breaking them up, but the PR is still very short. If you'd prefer though it could be split.

**Pull request merged**